<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerSchedule</class>
 <widget class="QWidget" name="LayerSchedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1" rowspan="2">
    <widget class="QStackedWidget" name="stackRight">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <widget class="QWidget" name="rPage0" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tb2">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rPage1">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="cb"/>
          </item>
          <item>
           <widget class="QLineEdit" name="txt"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnChangeBtn">
            <property name="text">
             <string>更改</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn0">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>800</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn1">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>900</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>1000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>1100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>1200</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>1300</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn6">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>1400</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn7">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>1500</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn8">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>1600</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn9">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>1700</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>36</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>←</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>243</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnName">
           <property name="styleSheet">
            <string notr="true">QPushButton::checked{
	color: rgb(255, 255, 255);
background-color: rgb(0, 170, 255);
}</string>
           </property>
           <property name="text">
            <string>人</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCost">
           <property name="styleSheet">
            <string notr="true">QPushButton::checked{
	color: rgb(255, 255, 255);
background-color: rgb(0, 170, 255);
}</string>
           </property>
           <property name="text">
            <string>$</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>→</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnSave">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>儲存</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QWidget" name="wBg" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stack">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSpinBox::up-button,QSpinBox::down-button
{
	width:0px;
}</string>
        </property>
        <widget class="QWidget" name="page0" native="true">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tb0">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>120</number>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <row>
             <property name="text">
              <string>白領11-19</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>白儲11-19</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>早班10-18</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>早班10-18</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>白班11-19</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>白班11-19</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>晚班18-0130</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>晚班18-0130</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>晚領19-03</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>晚儲19-02</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>夜儲20-03</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>夜班20-03</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>星期一</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>星期二</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>星期三</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>星期四</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>星期五</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>星期六</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>星期日</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page1">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tb1">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <property name="gridStyle">
             <enum>Qt::DashLine</enum>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="cornerButtonEnabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>120</number>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <row>
             <property name="text">
              <string>白領11-19</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>白儲11-19</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>早班10-18</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>早班10-18</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>白班11-19</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>白班11-19</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>晚班18-0130</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>晚班18-0130</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>晚領19-03</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>晚儲19-02</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>夜儲20-03</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>夜班20-03</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>星期一</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>星期二</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>星期三</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>星期四</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>星期五</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>星期六</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>星期日</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>btnSave</zorder>
  <zorder>wBg</zorder>
  <zorder>stackRight</zorder>
  <zorder>widget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
